<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMakeBuilderPreferences</class>
 <widget class="QWidget" name="CMakeBuilderPreferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>363</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label1">
     <property name="text">
      <string comment="@label:chooser">Default CMake executable:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="KUrlRequester" name="kcfg_cmakeExecutable">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label2">
     <property name="text">
      <string comment="@label:listbox">Default generator:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="kcfg_generator"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QWidget</extends>
   <header>KUrlRequester</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
